### PR TITLE
DisCatSharp

### PR DESCRIPTION
**Pull request by xHyroM**

### CODE REVIEW

**Comment by xHyroM** [comment]
@advaith1 🚀

File: libs.ts [hunk 1]
```diff
@@ -61,6 +61,20 @@ export const libs: Lib[] = [
 		contextMenus: 'Dev Version',
 		autocomplete: 'Dev Version'
 	},
+        {
```
Comment (Lulalaby) [inline libs.ts:64]:
```suggestion
    {
```

**Comment by advaith1** [comment]
@Lulalaby is the data correct?

**Comment by Lulalaby** [comment]
Yesu

**Pull request merged**